src/schemas/string.ts

```diff
@@ -43,14 +45,23 @@ export function walletAddrPatten(chainId: SupportedChainId) {
     case "42161":
     case "80002":
     case "421614":
+    case "10":
+    case "11155420":
+    case "8453":
+    case "84532":
       return evmAddrPattern;
     case "pisco-1":
     case "phoenix-1":
       return terraAddrPattern;
     case "juno-1":
     case "uni-6":
       return junoAddrPattern;
-
+    case "kaiyo-1":
```

Comment:
@SovereignAndrey, also need to set mainnet (as testnet is not supported in wallet connect ) cosmos chainId in `src/contexts/WalletContext/useKeplr/useKeplr.ts` and `src/contexts/WalletContext/wallet-connect/useKeplrWC.ts`

Reply:
Thanks for pointing that out. Fixed in c9eac899.